packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/LineGraph/BasicLineGraphs/DashedLine.6.doc.mdx

```diff
@@ -10,19 +9,16 @@ This is a Simple graph with dashed line in it.
 ## Getting Started
 
 - Install with [npmjs](https://www.npmjs.com):
-  - `npm install terra-graphs`
+  - `npm i @cerner/carbon-graphs --save-dev`
 
 ## Usage
-```jsx
-import React from 'react';
-import LineGraph from '@cerner/terra-graphs/lib/components/Line/LineGraph';
+```js
+import Carbon from "@cerner/carbon-graphs/dist/js/carbon-graphs.js";
+import "@cerner/carbon-graphs/dist/css/carbon-graphs.css";
```

Comment:
```suggestion
import Carbon from '@cerner/carbon-graphs/dist/js/carbon-graphs.js';
import '@cerner/carbon-graphs/dist/css/carbon-graphs.css';
```

Reply:
Fixed in this [commit](https://github.com/cerner/terra-graphs/pull/180/commits/69443571d46a4de8c3bc435a0c403d57c3203f98).